x/commons/wasm.go

```diff
@@ -0,0 +1,18 @@
+package commons
+
+import (
+	"encoding/json"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+)
+
+func HandleWasmMsg(cdc codec.Codec, data json.RawMessage, msg sdk.Msg) ([]sdk.Msg, error) {
```

Comment:
Can you add a few line of docs to this function?

Reply:
Added